.env.example

```
@@ -3,29 +3,39 @@
-# The selected profile in config/fleet.toml determines which declared provider
-# names are read; process exports win over this file.
-FLEET_DATABASE_URL=sqlite+aiosqlite:///./.fleet_rlm/local.sqlite3
-# Lakebase Postgres hosts the canonical schema. Never commit a real password.
+# The committed daytona-recursive profile routes Root and Sub through the
+# Modal-hosted GLM Chat Completions endpoint. To use another OpenAI-compatible
+# provider, update the profile's api_key_env and base_url_env in
+# config/fleet.toml. Process exports win over this file.
+#   FLEET_MODAL_BASE_URL=https://<modal-endpoint>.modal.direct/v1
+FLEET_MODAL_API_KEY=
+FLEET_MODAL_BASE_URL=
+FLEET_DAYTONA_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FLEET_DAYTONA_API_KEY key should go before the FLEET_MODAL_API_KEY key

(UnorderedKey)